composer.json

```diff
@@ -6,7 +6,7 @@
     "require": {
         "php": "^8.1",
         "ext-json": "*",
-        "monolog/monolog": "^2.0",
+        "monolog/monolog": "^2.0 || ^3.0",
```

Comment:
because you've used LogRecord this version does not support monolog ^2.0 anymore

Reply:
https://github.com/ETSGlobal/LogBundle/pull/26